src/components/app/Home.css

```diff
@@ -196,3 +196,9 @@
     padding-left: 0;
   }
 }
+
+@media (max-width: 767px) {
+  .homeSection {
+    width: 90%;
+  }
+}
```

Comment:
I think you can do much more than this:
1. the `max-width` property from the previous media query still applies and as a result I believe this property doesn't full apply.
2. you can remove some horizontal padding, for example set it to 1em.
3. you can make the "legal banner" take the full width as in your previous patch.
4. There's a problem with the vertical size of `<html>`, but I tried to fix it in the past and it was quite hard. The result of this is that the title usually disappears in "portrait" orientation. But this is difficult to fix without messing up the profile viewer... So maybe you can try to do it but in a separate PR